### PR TITLE
Added getter function to return the wrapped source within the cluster

### PR DESCRIPTION
**Pull request by acanimal**
Added a `getSource()` method to return the wrapped source within the cluster, in the same way as `ol.source.ImageVector`.
See #3267 and https://groups.google.com/forum/#!topic/ol3-dev/EfYFeskOqyc

### CODE REVIEW

File: src/ol/source/clustersource.js [hunk 1]
```diff
@@ -60,6 +60,15 @@ goog.inherits(ol.source.Cluster, ol.source.Vector);
 
 
 /**
```
Comment (elemoine) [inline src/ol/source/clustersource.js]: Every api function should have associated docstrings.

Comment (acanimal) [inline src/ol/source/clustersource.js]: Do you mean a description? It is a copy&paste from `ol.source.ImageVector`.

Comment (elemoine) [inline src/ol/source/clustersource.js]: Yes, a description. Likewise for ImageVector.

Comment (acanimal) [inline src/ol/source/clustersource.js]: Done. Check please if you agree or want a more detailed description.

**Comment by elemoine** [comment]
Thanks.

**Pull request merged**